.github/workflows/help-command.yml

```diff
@@ -0,0 +1,23 @@
+name: help-command
+
+on:
+  repository_dispatch:
+    types: [help-command]
+
+jobs:
+  help:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update comment
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          token: ${{ secrets.NEPTUNE_BOT_ACCESS_TOKEN }}
+          comment-id: ${{ github.event.client_payload.github.payload.comment.id }}
+          body: |
+            > Command | Description
+            > --- | ---
+            > /pre-commit-apply | Commit a pre-commit suggestions
+            > /run-unit | Run unittests on full matrix
```

Comment:
`/run-unit` - unit tests should run on each commit. Do we need command?

Reply:
On each commit we're running ubuntu x py3.7 + py.310 and that command will trigger unittests on full matrix (win,mac,linux x 3.7-3.10).

Reply:
Can you add the matrix to description? 
```
/run-unit | Run unittests on full matrix*
/run-e2e | Run E2E tests on full matrix*
*(win,mac,linux x 3.7-3.10)
```